From 447e1131280225b67507097186d1867a1fdac9e1 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 12 May 2021 12:32:05 +0300
Subject: [PATCH] Revert "[SW-39285] gaudi: security: gic privilege en bit"

This reverts commit b7f1769a2fe1aa13e3ca5fb4694bb9ac8ea0b93b.

This revert is done as the "gic privilege enable" status bit
is now set in ARMCP, Linux(constituents of gaudi-fit.itb). ARMCP
contains the implementation of irq polling mechanism.
So its better to keep the status bit enable code in the relevant
components to avoid version mismatch.

Change-Id: I99e4365d2087c65e5a5705dabd396d624974d604
---

diff --git a/gaudi/security.c b/gaudi/security.c
index 9c832e2..668eb12 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1613,7 +1613,6 @@
 	cpu_boot_dev_sts_clrset(CPU_BOOT_DEV_STS0_SECURITY_EN, is_sec_en);
 
 	cpu_boot_err_clrset(CPU_BOOT_ERR0_SECURITY_NOT_RDY, false);
-	cpu_boot_dev_sts_clrset(CPU_BOOT_DEV_STS0_GIC_PRIVILEGED_EN, true);
 	preboot_puts("Security init finished.\n");
 }
 
